doc/nrf/app_dev/board_names.rst

```diff
@@ -1,14 +1,23 @@
+.. _app_boards:
 .. _app_boards_names:
 .. _programming_board_names:
 
-Board names
-###########
+Board support
```

Comment:
Should the file name also be changed to board_support? For samples, the file name follows the Title name.

Reply:
I won't change the file name to avoid breaking the file edit history. I also suspect we might need this section back in the future if more than 1 page qualifies for it.